src/index.js

```diff
@@ -0,0 +1,31 @@
+import './style.css';
+import reloadIcon from './refresh.png';
+import enterIcon from './enter.png';
+import moreIcon from './more.png';
+
+const refresh = document.querySelector('.refresh');
+refresh.src = reloadIcon;
+const enter = document.querySelector('.enter');
+enter.src = enterIcon;
+const more = document.querySelectorAll('.more');
+more.forEach((img) => {
+  img.src = moreIcon;
+});
+
+const tasks = [
+  {
+    description: '',
+    completed: false,
+    index: 0,
+  },
+];
+
+tasks.forEach((task) => {
+  const list = document.createElement('div');
+  list.classList.add('list-item');
+  list.innerHTML = `
+  <input type="checkbox">
+  <p>${task.description}</p>
+  <img src="" alt="">
+  `;
+});
```

Comment:
Well done on this project 💯 but I notice here that you're going through the array `tasks` but you're not populating the HTML list item element for each task. Consider adding that as it will improve both your code and your project. 😉